packages/e2e/package.json

```diff
@@ -24,6 +24,7 @@
     "@aws-amplify/ui": "^5.3.1",
     "@badeball/cypress-cucumber-preprocessor": "^12.1.0",
     "@bahmutov/cypress-esbuild-preprocessor": "^2.1.3",
+    "@nuintun/qrcode": "^3.3.0",
```

Comment:
@calebpollman I believe a cypress test, testing the QR code generated, and looking at the image is the best way of doing this. And since it's a dev dependency, for the Cypress e2e test, the impact is minimal.

Reply:
While i appreciate that you feel that it is the best way, is there any other way to test the output of the QR code without adding a 3p dep?

Reply:
Not that I know of.

Reply:
Our existing QRCode library does not decode, -> https://github.com/soldair/node-qrcode/issues/129